src/customize-pop/accessibility-seeing.md

```diff
@@ -0,0 +1,187 @@
+# Seeing
+
+`Seeing` settings provide visual enhancements to help visually impaired users navigate the operating system's user interface.
+
+## High Contrast
+
+Enabling `High Contrast` applies a high contrast theme to windows and icons.
+
+### High Contrast Mode Off
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/contrast-mode-off.png" target="blank">
+    <img src="/images/accessibility-settings/contrast-mode-off.png" width="733px" height="456px">
+</a>
+
+### High Contrast Mode On
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/contrast-mode-on.png" target="blank">
+    <img src="/images/accessibility-settings/contrast-mode-on.png" width="733px" height="456px">
+</a>
+
+## Large Text
+
+Enabling `Large Text` increases the size of text across system windows and supported applications.
+
+### Large Text Disabled
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/before-lt.png" target="blank">
+    <img src="/images/accessibility-settings/before-lt.png" width="733px" height="456px">
+</a>
+
+### Large Text Enabled
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/after-lt.png" target="blank">
+    <img src="/images/accessibility-settings/after-lt.png" width="733px" height="456px">
+</a>
+
+## Enable or Disable Animations
+
+Animations appear when minimizing and maximizing windows. Disabling animations may improve performance on systems with low video resources.
+
+### Animations Enabled
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/animations-enabled.webm" />
+</video>
+
+### Animations Disabled
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/animations-disabled.webm" />
+</video>
+
+## Cursor Size
+
+`Cursor Size` allows users to choose between five cursor sizes.
+
+![Cursor Size](/images/accessibility-settings/cursor-size.png)
+
+## Zoom
+
+The `Zoom` setting designates all or a part of the screen to display magnified content. This feature can be further customized to render the zoomed area specific portions of the screen, increase or decrease the zoom magnification, add a crosshair, and add color effects to the zoomed area.
```

Comment:
> This feature can be further customized to render the zoomed area specific portions of the screen,

I'm not entirely sure what this means, but it sounds like it's missing a word. Is it supposed to be "to specific portions of the screen" or "of specific portions of the screen"?

Reply:
"to specific portions of the screen", was missing a word, updated.